apk-generator/index.html

```diff
@@ -121,13 +121,19 @@ <h3 id="status"></h3>
                 $('#generator-progress').css('display', 'block')
                 $('#upload-progress').css('display', 'block')
                 var ary = $(form).serializeArray();
+                var emailse = $("#Email").val();
                 var obj = {};
                 updatePercentUpload(10);
                 for (var a = 0; a < ary.length; a++) obj[ary[a].name] = ary[a].value;
                 console.log("JSON", obj);
                 if (obj.Email == "" || obj.App_Name == "") {
```

Comment:
Ok sounds good. :+1:
Also , just noticed that it was elseif.
I was thinking that it was if the whole time.
looks good, proceed ahead.